### DISCUSSION TITLE
Write column content to file

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#1750] [Q&A] [answered]
I used VisiData to open an HTML sheet. I want to take all values off one specific column and write them to a file, newline-separated. Any idea how to do that as quickly as possible?

**Comment by saulpw** [comment, marked as answer]
See `File > Save > current column`.

**Reply by cool-RR** [reply]
Great, thank you. My next question was going to be how to save it without the header, and I see that choosing a `.txt` extension does that.
